MaterialLib/MPL/Properties/CapillaryPressureSaturation/CapillaryPressureVanGenuchten.h

```diff
@@ -19,56 +17,73 @@ class Medium;
 class Phase;
 class Component;
 
-/// \copydoc MaterialPropertyLib::SaturationVanGenuchten
+/**
+ * \brief The van Genuchten capillary pressure model.
```

Comment:
Why not use \copydoc MaterialPropertyLib::SaturationVanGenuchten or use it  together with some additional words?

Reply:
This is not a part of new property. If something needs to be changed across different code parts, it should be done in a different PR.

Reply:
If you want to keep the full documentation here. I suggest to use \copydoc  in MaterialPropertyLib::SaturationVanGenuchten to avoid duplication (could be in another PR.).

Reply:
That's absolutely fine to have a single page describing the van Genuchten capillary pressure - saturation. In another PR.